enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/security/jwt/validator/JWTValidator.java

```diff
@@ -66,21 +64,21 @@ public JWTValidationInfo validateJWTToken(SignedJWTInfo signedJWTInfo) throws En
 
         if (StringUtils.isNotEmpty(issuer) && tokenIssuers.containsKey(issuer)) {
             ExtendedTokenIssuerDto tokenIssuer = tokenIssuers.get(issuer);
-            this.jwtTransformer = ConfigHolder.getInstance().getConfig().getJwtTransformerMap().get(issuer);
-            if (this.jwtTransformer == null) {
-                this.jwtTransformer = new DefaultJWTTransformer();
+            JWTTransformer jwtTransformer = ConfigHolder.getInstance().getConfig().getJwtTransformerMap().get(issuer);
+            if (jwtTransformer == null) {
+                jwtTransformer = new DefaultJWTTransformer();
```

Comment:
It seems we can avoid instantiating jwtTransformer for each request.

Reply:
improved with https://github.com/wso2/product-microgateway/pull/2823/commits/a46a8cf968bb82a372c7fac56d24977f61bd4b54